server/lib/src/valueset/eckey.rs

```diff
@@ -99,7 +99,12 @@ impl ValueSetT for ValueSetEcKey {
     }
 
     fn generate_idx_eq_keys(&self) -> Vec<String> {
-        Vec::with_capacity(0)
+        let key_der = match self {
```

Comment:
Do we actually even need idx keys here? I don't think we will be able to query this type, I think we return an empty vec.

Reply:
yeah I added it at 2am because it was throwing some `InvalidSyntax` error on startup so I started doing random shit